src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -10,11 +10,16 @@ internal class LinuxAppServiceEventGenerator : LinuxEventGenerator
     {
         private readonly LinuxAppServiceFileLoggerFactory _loggerFactory;
         private readonly HostNameProvider _hostNameProvider;
+        private readonly bool _consoleEnabled = true;
 
         public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFactory, HostNameProvider hostNameProvider)
         {
             _loggerFactory = loggerFactory;
             _hostNameProvider = hostNameProvider ?? throw new ArgumentNullException(nameof(hostNameProvider));
+            if (Environment.GetEnvironmentVariable(EnvironmentSettingNames.ConsoleLoggingDisabled) == "1")
```

Comment:
Is this check expected on Linux dedicated?

Reply:
Some background on this appsetting: It was added for investigating performance impact of different logging mechanisms in consumption. So not needed in dedicated.